src/SimpleCalc/SimpleCalc.csproj

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project DefaultTargets="Build" ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
+    <Platform Condition=" '$(Platform)' == '' ">anycpu</Platform>
+    <ProjectGuid>{A9953C27-C02B-4599-8B4D-0DDC0C5216AA}</ProjectGuid>
+    <OutputType>Exe</OutputType>
+    <RootNamespace>SimpleCalc</RootNamespace>
+    <AssemblyName>SimpleCalc</AssemblyName>
+    <TargetFrameworkVersion>v4.7.2</TargetFrameworkVersion>

```

Comment:
We created .NET Core apps, not .NET Framework, be attentive..

Reply:
Oo, many thanks for attentiveness @MikhailMasny ! It was a kind of fatal miss-click :)